.github/workflows/publish.yml

```diff
@@ -85,18 +85,28 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+      - name: Install dependencies
+        run: |
+          poetry install -E all -vv
       - name: Build image
         run: |
           cd $( dirname ${{ matrix.dockerfile.path }})
           VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
           sed -i "s#etna\[all\]#etna\[all\]==$VERSION#g" requirements.txt
+          poetry export --without-hashes --format=requirements.txt >> requirements.txt
```

Comment:
I should rework this place. It seems like this requirements doesn't contain all dependencies for "all" extras.